packages/flame/lib/src/test_helpers/mock_canvas.dart

```diff
@@ -115,6 +118,34 @@ class MockCanvas extends Fake implements Canvas, Matcher {
     return mismatchDescription.add(matchState['description'] as String);
   }
 
+  bool containsAtLeast(covariant MockCanvas other, Map matchState) {
```

Comment:
since everything was private, I figured having this here was the easiest way to implement a different Matcher using simple inheritance, rather than composition. but I am not super familiar on how to re-use and combine these Matchers, so open to suggestions for other implementations